src/components/urls.ts

```diff
@@ -0,0 +1,14 @@
+export const HOME_PAGE = '/';
```

Comment:
Maybe let's call that file `routes.ts`

HOME_ROUTE
LOGIN_ROUTE

What do you think ? it's easier to understand I think at first sight

Reply:
We can go with _ROUTE and change URLs -> routes or _URL and keep the URLs file